### PR TITLE
run watch commands when starting Gitpod

### LINKED ISSUE #7497: Proposal: run watch commands when starting Gitpod
Goal: make it even easier for someone to update css/js in Gitpod

I would like to automatically run the watch commands in gitpod:
- `docker-compose run --rm home npm run-script watch`
- `docker-compose run --rm home npm run-script watch-css`

### Describe the problem that you'd like solved

The few times I pop over to Open Library to try to make a small css change or see if I can fix something I always have to dig around and see what command I need to run to get the CSS to update.
I think it would be nice for the dev environment in Gitpod to automatically run the watch commands so any changes made are picked up right away.

### Proposal & Constraints

These can be done by having separate tasks that start after we have docker running.

### Additional context

Happy to open a PR for this.

I consider this a blocker for https://github.com/internetarchive/openlibrary/issues/6939
Or at least a nice to have

### Stakeholders

@cclauss @mekarpeles

### PR DESCRIPTION
**Pull request by RayBB**
Closes #7497

Automatically runs the watch commands when starting in Gitpod.

Feel free to edit this PR as needed before merging 👍 

PS: After this I'll work on #6939 :)

### Technical

I didn't use concurrently because I didn't see examples of how to do that and I wasn't clear how to do that.

If someone would like to propose a nicer way to do this I'm open to it. But I figured this is a good point to start talking from.

### Testing

Open this PR in gitpod and try making changes to css/js.

### Screenshot

https://user-images.githubusercontent.com/921217/222598532-c8037bb3-6277-48df-9dbc-fc6fcc99c116.mp4

### Stakeholders

@cdrini

### CODE REVIEW

**Comment by RayBB** [comment]
A small bump to get this reviewed. @cdrini @cclauss

File: .gitpod.yml [hunk 1]
```diff
@@ -10,6 +10,12 @@ ports:
 - port: 3000
   onOpen: ignore
 tasks:
+- name: watch-js
+  # This task watches for changes to JS files and automatically builds
+  init: gp sync-await docker-up && docker-compose run --rm home npm run-script watch
+- name: watch-css
+  # This task watches for changes to CSS files and automatically builds
+  init: gp sync-await docker-up && docker-compose run --rm home npm run-script watch-css
```
Comment (cdrini) [inline .gitpod.yml:18]: Can you rebase and test if this works? Concurrently is now in the package.json :)

```suggestion
- name: watch
  # This task watches for changes to JS/CSS files and automatically builds
  init: gp sync-await docker-up && docker-compose run --rm home npx concurrently npm:watch npm:watch-css
```

If it errors, might need to wait for the next deploy until the olbase image is regenerated

**Review by cdrini — APPROVED** [review]
Tested works like a charm! JS and CSS now auto-rebuild.

**Comment by RayBB** [comment]
Thanks for following up on this. I let it slip through the cracks. Happy to see something nice merged in!

**Comment by cdrini** [comment]
No worries 😊 Thank you for your patience! It got stuck in my backlog for quite a while!

**Pull request merged**